server/team_matchmaker/player_party.py

```diff
@@ -0,0 +1,43 @@
+from server.players import Player
+
+
+class PlayerParty:
+    def __init__(self, team_matchmaking_service: "TeamMatchmakingService", owner: Player):
+        self.team_matchmaking_service = team_matchmaking_service
+        self._members = {owner}
+        self.owner = owner
+
+    @property
+    def members(self):
+        return frozenset(self._members)
+
+    def add_player(self, player: Player):
+        self._members.add(player)
+        self.team_matchmaking_service.player_parties[player] = self
+
+        self.broadcastParty()
+
+    def remove_player(self, player: Player):
+        self._members.remove(player)
+        self.team_matchmaking_service.player_parties.pop(player)
+
+        self.broadcastParty()
+        self.sendParty(player)
+
+    def broadcastParty(self):
+        for player in self._members:
+            self.sendParty(player)
+
+    def sendParty(self, player: Player):
```

Comment:
Please consistently use `snake_case` and do not start intermixing `camelCase`

Reply:
You don't want to know how often I caught myself using camelcase ;), it's just natural to me and I don't notice it.